### PR TITLE
PDI-48 no additional work required as this was blocked due to rounding error…

### PR DESCRIPTION
**Pull request by DaveGill-Nimbus**
…s which have been fixed in the DAS pipeline.

Change simply uncomments the tag in the test so it'll run.

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Additional Payments English and Maths/Non-Levy learner funding agreed within maximum completes maths and english early PV2-485.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿#@supports_dc_e2e currently blocked by PV2-1014
+﻿@supports_dc_e2e currently blocked by PV2-1014
 Feature: Non-levy learner completes maths and english early PV2-485
```
Comment (CarlHA) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Additional Payments English and Maths/Non-Levy learner funding agreed within maximum completes maths and english early PV2-485.feature]: Please remove comment after tag

Comment (DaveGill-Nimbus) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Additional Payments English and Maths/Non-Levy learner funding agreed within maximum completes maths and english early PV2-485.feature]: removed and pushed

**Comment by CarlHA** [comment]
Replaced by #502

**Comment by CarlHA** [comment]
No longer required as it has been resolved and merged into PDI-Sprint1

**Pull request closed without merging**